docs/topic-guides/blendable-web-apps/localization.md

```diff
@@ -0,0 +1,115 @@
+# Localizing Starcounter Apps
+
+Real world applications often need to support a variety of languages. This article describes how to add such support to your application. It doesn't describe any practices for translation of texts or internals of how localization support works in Starcounter.
+
+## Getting started
+
+1. Create resources file with strings you want to localize.
+   1. In Visual Studio, open "Add new item" menu and choose "Resources File"
+   1. Select any name for your file. We'll use `Strings.resx` in this example
+   1. Add as many resources files as many languages you want to support. Use the following format when naming them: `name.ll-cc.resx` where `ll` is the language code and `cc` is the country code. To continue our example, you would add `Strings.sv-se.resx` to support Swedish language as used in Sweden.
+1. Populate your resources files with texts you want to localize.
+   1. The `name` column should contain a valid C# indentifier, like `SignIn`. Value should be the relevant text in language chosen for this file. For example, it could be "Logga in" in file `Strings.sv-se.resx`. The comments column can be used freely.
+1. Visual Studio will generate a class from your resources file. In our example, it will generate `Strings` class with a string member `SignIn`
+   1. Value of this member will depend on currently selected culture. You don't have to worry about setting it in your application, it's done by a special language selection application.
```

Comment:
"The value of this member". Also, are they always properties or can they also be fields? If they're always properties, I would just write "property" instead of "member".

Reply:
always properties